content/en/docs/Policy Reports/_index.md

```diff
@@ -97,56 +136,12 @@ kubectl get clusterpolicyreport
 ```
 
 {{% alert title="Tip" color="info" %}}
-For a graphical view of Policy Reports, check out [Policy Reporter](https://github.com/kyverno/policy-reporter#readme).
+Note that the name of the report is mostly random, don't forget to add `-o wide` to  show additional informations that will help identify the resource associated with the report.
 {{% /alert %}}
 
-## Viewing policy violations
```

Comment:
Rather than deleting this section altogether, can you adapt it so that you show the command on how to retrieve failing results from a single PolicyReport? I think that's still really valuable, and I worked hard figuring out how to write that JSONPath pipeline :)

Reply:
Ok, but just for the sake of your pipeline then....